WalletWasabi/Wallets/Wallet.cs

```diff
@@ -39,56 +39,61 @@ public class Wallet : BackgroundService
 	{
 		private Node _localBitcoinCoreNode = null;
 
-		public Wallet(
-			Network network,
+		public Wallet(string dataDir, Network network, string filePath) : this(dataDir, network, KeyManager.FromFile(filePath))
+		{
+		}
+
+		public Wallet(string dataDir, Network network, KeyManager keyManager)
```

Comment:
It seems the `Wallet` class is created as a simple container for a `KeyManager` but it can became a useful wallet once the services are registered and the Wallet is started. IMO this is not a good design, the other dependencies should be initialized in the constructor too (except the subscriptions to the events which could be subscribed on the Start and unsubscribe in the Stop.)

Reply:
It is useful, immediately after the creation, as we have a constant reference to one specific logical object which describes a wallet. 
For example, I can have a list of the wallets and I can just call start from the UI, without adding business logic there as we had before. 

I can pass the services when Walletmanager.RegisterServices called, but in that case, I had to separate the things like subscriptions - with this PR I want to minimize the refactorings and stay with the original logic, which similar to this. WalletService created and subscribed just before the Start.